keps/sig-network/3705-cloud-node-ips/README.md

```diff
@@ -265,138 +182,6 @@ cloud-provider-specific changes should be needed; we should be able to
 make the needed changes in the `cloud-provider` module, and then the
 individual providers just need to revendor to the new version.
 
-### Example of `--node-ip` possibilities
```

Comment:
Hmm - I actually really liked this table below - why you're removing it?

Reply:
I can put it back... (I removed it because a lot of it was to show how `--node-ip IPv6` was different from `--node-ip ::`, which is no longer part of the KEP.)